.github/ISSUE_TEMPLATE/Patch-Version-Release-Checklist.md

```diff
@@ -0,0 +1,135 @@
+
+> ⚠️ **Warning:** The release checklist is a living document. It will be updated as the release progresses. Please check back often to ensure that you are up to date with the latest information.
+
+## Checklist
+- [ ] Codebase Stability
+- [ ] Breaking Changes Validation
+- [ ] Compile Changelog
+- [ ] Release
+- [ ] Release Announcement
+
+# Release Details
+The release manager is responsible for completing the release end-to-end ensuring that all the necessary steps are taken and that the release is completed in a timely manner. This is usually the owner of the release issue but may be assigned to a different maintainer if necessary.
+
+- Release Manager: <!-- Insert Release Manager Here -->
+- Release Date: <!-- Insert Release Date Here -->
+- Release Branch: `main`
+
+# Release Steps
+The release manager is expected to execute the checklist below. The release manager is also responsible for ensuring that the checklist is updated as the release progresses. Any changes or issues should be communicated under this issue for centralized tracking.
+
+## 1. Codebase Stability
+- [ ] The `main` branch build passes
+
+    [![Build Status](https://github.com/celery/celery/actions/workflows/python-package.yml/badge.svg)](https://github.com/celery/celery/actions/workflows/python-package.yml)
+
+## 2. Breaking Changes Validation
+A patch release should not contain any breaking changes. The release manager is responsible for reviewing all of the merged PRs since the last release to ensure that there are no breaking changes. If there are any breaking changes, the release manager should discuss with the maintainers to determine the best course of action if an obvious solution is not apparent.
+
+## 3. Compile Changelog
+The release changelog is set in two different places:
+1. The [Changelog.rst](https://github.com/celery/celery/blob/main/Changelog.rst) that uses the RST format.
+2. The GitHub Release auto-generated changelog that uses the Markdown format. This is auto-generated by the GitHub Draft Release UI.
+
+> ⚠️ **Warning:** The pre-commit changes should not be included in the changelog.
+
+To generate the changelog automatically, [draft a new release](https://github.com/celery/celery/releases/new) on GitHub using a fake new version tag for the automatic changelog generation. Notice the actual tag creation is done **on publish** so we can use that to generate the changelog and then delete the draft release without publishing it thus avoiding creating a new tag.
+
+- Create a new tag
+<img width="502" alt="CleanShot 2023-09-05 at 22 06 24@2x" src="https://github.com/celery/celery/assets/4662342/69d7eb78-0dd1-4a37-9266-d718e2340fad">
+
+- Generate Markdown release notes
+<img width="389" alt="CleanShot 2023-09-05 at 22 13 39@2x" src="https://github.com/celery/celery/assets/4662342/83c6a4b6-1676-496e-8d75-5a6b10352b5b">
+
+- Copy the generated release notes.
```

Comment:
this this counter intuitive to me

Reply:
I know it's a "dummy level" checklist but the point is avoiding human-error.
Do you want me to change this then? @auvipy